x-pack/platform/plugins/shared/sample_data_ingest/server/services/sample_data_manager/sample_data_manager.ts

```diff
@@ -136,6 +136,6 @@ export class SampleDataManager {
   }
 
   private getSampleDataIndexName(sampleType: DatasetSampleType): string {
-    return `${this.indexPrefixName}-${sampleType.toLowerCase()}`;
+    return `${this.indexPrefixName}${sampleType.toLowerCase()}`;
```

Comment:
given the `sampleType` is enum with the docs name currently wondering if we should add a prefix or suffix for docs?

```suggestion
    return `${this.indexPrefixName}documentation_${sampleType.toLowerCase()}`;
```

Personally I think it would be good for the final index name to be a bit more descriptive. so `kibana_sample_data_documentation_elasticsearch` instead of just `kibana_sample_data_elasticsearch`. 

actually `kibana_sample_data_elasticsearch_documentation` might be better.

Currently `DatasetSampleType` is only one value, but not sure if we'd want to make that change in the enum so we could have `DatasetSampleType`s there were not documentation.